packages/components/src/snackbar/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { flow, noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '../';
+
+const stopPropagation = ( event ) => event.stopPropagation();
+const NOTICE_TIMEOUT = 10000;
+
+function Snackbar( {
+	className,
+	children,
+	actions = [],
+	onRemove = noop,
+} ) {
+	useEffect( () => {
+		// This rule doesn't account yet for React Hooks
+		// eslint-disable-next-line @wordpress/react-no-unsafe-timeout
+		const timeoutHandle = setTimeout( () => {
+			onRemove();
+		}, NOTICE_TIMEOUT );
+
+		return () => clearTimeout( timeoutHandle );
+	}, [] );
+
+	const classes = classnames( className, 'components-snackbar' );
+
+	return (
+		<div
+			className={ classes }
+			onClick={ onRemove }
+			tabIndex="0"
```

Comment:
Those notices are announced with `wp.a11y.speak` method like all other types of notices. They are also automatically being dismissed after 10 seconds. Do they really need to be focusable? What are the odds that keyboard-only user will want to tab to this message to dismiss it before it gets removed?

Reply:
> Pressing the ESC key while one of the snackbar’s child elements has focus (e.g., the action button) will dismiss the snackbar.

Yes, no strong opinions from me but I had this feedback earlier.